src/zeit/locales/zeit.cms.pot

```diff
@@ -4765,10 +4761,6 @@ msgid "Hide timestamp"
 msgstr ""
 
 #: zeit/seo/interfaces.py:44
-msgid "Disable intext links"
-msgstr ""
```

Comment:
Das scheint zumindest noch in der angegebenen Datei verwendet zu werden. Vielleicht doch lieber behalten.